test/e2e/run.sh

```diff
@@ -748,8 +759,10 @@ vm=${vm:=$(basename ${TOPOLOGY_DIR})-${distro}-${cri}}
 vm_files=${vm_files-""}
 cri_resmgr_cfg=${cri_resmgr_cfg-"${SCRIPT_DIR}/cri-resmgr-memtier.cfg"}
 cri_resmgr_extra_args=${cri_resmgr_extra_args-""}
+cri_resmgr_agent_extra_args=${cri_resmgr_agent_extra_args-""}
 cleanup=${cleanup-0}
 reinstall_cri_resmgr=${reinstall_cri_resmgr-0}
+py_consts="${py_consts-''}"
```

Comment:
nitpick: same nitpick here as for earlier patches regarding `:-` vs. `-`

Reply:
Switched to `:-`.